src/main/java/uk/gov/hmcts/reform/iacaseapi/infrastructure/security/CcdEventAuthorizor.java

```diff
@@ -1,15 +1,20 @@
 package uk.gov.hmcts.reform.iacaseapi.infrastructure.security;
 
+import static org.slf4j.LoggerFactory.getLogger;
+
 import java.util.Collections;
 import java.util.List;
 import java.util.Map;
 import java.util.stream.Collectors;
+
 import org.springframework.security.access.AccessDeniedException;
 import uk.gov.hmcts.reform.iacaseapi.domain.UserDetailsProvider;
 import uk.gov.hmcts.reform.iacaseapi.domain.entities.ccd.Event;
 
 public class CcdEventAuthorizor {
 
+    private static final org.slf4j.Logger log = getLogger(CcdEventAuthorizor.class);
```

Comment:
Could we keep the variable to capitals to keep it consistent? LOG instead of log?